### PR TITLE
Added test for SearchResultExtractorTrait

### PR DESCRIPTION
**Pull request by MarioBlazek**

### CODE REVIEW

**Comment by MarioBlazek** [comment]
ping @pspanja @emodric

File: tests/lib/Integration/Traits/SearchResultExtractorStub.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+use eZ\Publish\API\Repository\Values\Content\Search\SearchResult;
+
+class SearchResultExtractorStub
+{
+    use SearchResultExtractorTrait;
+
+    /**
+     * @param \eZ\Publish\API\Repository\Values\Content\Search\SearchResult $searchResult
+     *
+     * @return
```
Comment (pspanja) [inline tests/lib/Integration/Traits/SearchResultExtractorStub.php]: Missing return type hint

**Review by pspanja — COMMENTED** [review]
I have only one minor remark, otherwise looks good 👍

**Comment by MarioBlazek** [comment]
@pspanja return type added.

**Comment by pspanja** [comment]
Thanks @MarioBlazek :)

**Pull request merged**